core/modules/replica/HttpExceptions.h

```diff
@@ -65,6 +65,21 @@ class HttpError: public std::runtime_error {
     nlohmann::json const _errorExt = nlohmann::json::object();
 };
 
+
+/**
+ * Report an error as an exception.
+ * @note This method always throws and it never returns control back to a caller.
+ *   The method is meant to be used by Ingest system workers to report error conditions
+ *   that won't require aborting a transaction.
+ * @param scope A scope of the error.
+ * @param error A human readable error message.
+ * @param (optional) HTTP code of a error if applies.
+ * @throws HttpError
+ */
+void retryAllowedError(std::string const& scope,
```

Comment:
MAybe call this method as `raiseRetryAllowedError`, leading `retry` reads like an action verb.

Reply:
Good point! I'll rename the function.